src/UI/Internal/SideBar.elm

```diff
@@ -123,36 +182,49 @@ viewHead cfg (Menu.Menu prop _) label maybeStack =
                 label
 
 
-viewSide : RenderConfig -> Bool -> Menu msg -> Element msg
-viewSide cfg proportional (Menu.Menu prop opt) =
+viewSide : RenderConfig -> SidebarConfig -> Menu msg -> Element msg
+viewSide cfg config (Menu.Menu prop opt) =
     let
         adaptWidth =
             if prop.isExpanded then
-                if proportional then
+                if config.proportional then
                     width (fillPortion 75)
 
                 else
-                    width (px 228)
+                    width (px 264)
 
             else
                 width shrink
 
-        adaptHeader =
+        toggleMsg =
+            prop.toggleMsg (not prop.isExpanded)
+
+        header =
             if prop.isExpanded then
-                headerView
+                headerView cfg toggleMsg opt.logo
 
             else
-                slimHeaderView
+                slimHeaderView cfg toggleMsg Palette.grayLight1
     in
     Element.column
         [ height fill
         , adaptWidth
-        , Background.color Colors.gray.lightest
+        , if prop.isExpanded then
+            paddingXY 16 24
+
+          else
+            paddingXY 6 22
+        , Background.color <|
+            if config.persistent then
+                Colors.gray.lightest
+
+            else
+                Colors.white
```

Comment:
Checking figma, I think the menu has the same color when it is expanded, figma screenshot:

![Screen Shot 2021-02-01 at 11 09 18](https://user-images.githubusercontent.com/1559013/106444421-06f04a80-647e-11eb-8187-9424d72a65e5.png)